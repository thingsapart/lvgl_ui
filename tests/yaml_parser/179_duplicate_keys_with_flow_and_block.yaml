# A mix of styles resulting in duplicate keys
key: { value: flow }
key:
  value: block
