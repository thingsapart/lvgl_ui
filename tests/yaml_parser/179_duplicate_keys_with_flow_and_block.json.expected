[{
		"key":	{
			"value":	"flow"
		},
		"key":	{
			"value":	"block"
		}
	}]
